streams/next.json

```diff
@@ -190,226 +190,225 @@
-                                "sha256": "a7c4ce80aba34ac8c14d7f14490fd70459a32bbd947391a7f679b990b3ac2c56"
+                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/36.20220522.1.0/x86_64/fedora-coreos-36.20220522.1.0-live-rootfs.x86_64.img",
+                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/36.20220522.1.0/x86_64/fedora-coreos-36.20220522.1.0-live-rootfs.x86_64.img.sig",
+                                "sha256": "db123fcccde4ee411faf7f446c0416fccf2fba672b524772999ed43668d4948e"
                             }
                         },
                         "raw.xz": {
                             "disk": {
-                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/36.20220507.1.0/x86_64/fedora-coreos-36.20220507.1.0-metal.x86_64.raw.xz",
-                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/36.20220507.1.0/x86_64/fedora-coreos-36.20220507.1.0-metal.x86_64.raw.xz.sig",
-                                "sha256": "91f918de35db779f9fcb50353fe0559b0be8cd3149e3afa8f8e40ef94ec658c6",
-                                "uncompressed-sha256": "821e7e4227ddd64f5d873f954462e0c20b6f490770f65363970a5558ad3a4af6"
+                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/36.20220522.1.0/x86_64/fedora-coreos-36.20220522.1.0-metal.x86_64.raw.xz",
+                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/36.20220522.1.0/x86_64/fedora-coreos-36.20220522.1.0-metal.x86_64.raw.xz.sig",
+                                "sha256": "a83431816f50734cc272fd6680467dbbbf1c1169886114871d45d1f92d88caed",
+                                "uncompressed-sha256": "ee53cd36eaf80b9a1fecd429b8f8795f36a3425c06408f877297e3e5f3e90e1c"
                             }
                         }
                     }
                 },
                 "nutanix": {
-                    "release": "36.20220507.1.0",
+                    "release": "36.20220522.1.0",
                     "formats": {
-                        "qcow2.xz": {
+                        "qcow2": {
                             "disk": {
-                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/36.20220507.1.0/x86_64/fedora-coreos-36.20220507.1.0-nutanix.x86_64.qcow2.xz",
-                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/36.20220507.1.0/x86_64/fedora-coreos-36.20220507.1.0-nutanix.x86_64.qcow2.xz.sig",
-                                "sha256": "72dbbf0ec84715a06c4e74b78c276567d47a5f04fbe0083ea61961f8aa6fd9eb",
-                                "uncompressed-sha256": "116f582f0040e7b6b776eee474d2a1e8c21dd2a71257656ca1f1f3a40d6576e4"
```

Comment:
this unexpected change is expected. We no longer have an `uncompressed-sha256` because the artifact isn't externally compressed. See https://github.com/coreos/coreos-assembler/pull/2848